README.md

```diff
@@ -19,26 +19,40 @@
 It boots the device with multiple patches required. On first run, it'll boot a ramdisk which dumps your onboard blob, creates a fakefs (if using semi-tethered), installs the loader app, and patches your kernel. 
 
 # Warnings
-- We are **NOT** responsible for any data loss, or the result of a device being bricked. The user of this program accepts responsibility should something happen to their device. While nothing should happen, jailbreaking has risks in itself. If your device refuses to boot after using palera1n please use `iTunes` or `Finder` to restore iOS (or if you used tethered, use `--restorerootfs` at the end of your palera1n command).
+- We are **NOT** responsible for any data loss, or the result of a device being bricked. The user of this program accepts responsibility should something happen to their device. While nothing should happen, jailbreaking has risks in itself.
 
-- palera1n will never work in VirtualBox, VMware or any virtual machine that doesn't support a PCI passthrough.
+	- If your device refuses to boot after using palera1n, please use `iTunes` or `Finder` (or, if you have blobs, `futurerestore` or `idevicerestore`) to restore iOS (or, if you used tethered, [restore rootfs](https://ios.cfw.guide/removing-palera1n/)).
+
+- palera1n will never work in VirtualBox, VMware or any virtual machine that doesn't support PCI passthrough.
 
 # Requirements
-- Needs a checkm8 vulnerable iOS device on iOS 15.x or 16.x (`A8` - `A11`)
+- A checkm8 vulnerable iOS device on iOS 15.x or 16.x (`A8` - `A11`)
 	-	If you want the device to be semi-tethered, **you will need 5-10GB of space** for the fakefs. This means that 16GB devices cannot be semi-tethered
 	- If you are on `A10(X)`, use [checkp4le](https://github.com/guacaplushy/checkp4le) instead for full SEP functionality (Passcode, TouchID, Apple Pay)
```

Comment:
to make this better you could change checkp4le to checkra1n-c (as checkp4le is archived)